extensions/amp-analytics/0.1/test/test-variables.js

```diff
@@ -57,6 +57,7 @@ describe('amp-analytics.VariableService', function() {
     const vars = {
       '1': '1${2}', '2': '2${3}', '3': '3${4}', '4': '4${1}', '5': 0,
       'a': '${b}', 'b': '${c}', 'c': 'https://www.google.com/a?b=1&c=2',
+      'obj': {a: 1, b: 'xyz'}, 'array': [1, 'xyx', 2],
```

Comment:
Does this still work if you go one level deeper? 

```js
foo: {bar: 1}
obj: {a: ${foo}}
```

Reply:
nope. we don't expand nested object

Reply:
ok sgtm.